test/phonons.jl

```diff
@@ -5,110 +5,104 @@ using LinearAlgebra
 using ForwardDiff
 using StaticArrays
 
+@testset "Phonons" begin
+
 # Convert back and forth between Vec3 and columnwise matrix
 fold(x)   = hcat(x...)
 unfold(x) = Vec3.(eachcol(x))
 
 function prepare_system(; n_scell=1)
-    positions = [[0.,0,0]]
+    positions = [[0.0, 0.0, 0.0]]
     for i in 1:n_scell-1
-        push!(positions, i*ones(3)/n_scell)
+        push!(positions, i * ones(3) / n_scell)
     end
 
     a = 5. * length(positions)
     lattice = a * [[1 0 0.]; [0 0 0.]; [0 0 0.]]
 
-    s = DFTK.compute_inverse_lattice(lattice)
-    directions = [[s * [i==j,0,0] for i in 1:n_scell] for j in 1:n_scell]
+    L⁻¹ = DFTK.compute_inverse_lattice(lattice)
```

Comment:
Not super fan of this, it renders badly eg on android phones, and doesn't really help much compared to Linv (which we use at other places I believe)